src/workerd/api/http.h

```diff
@@ -543,12 +545,33 @@ struct RequestInitializerDict {
   // to accept `null` as an input, so if we're doing what Chrome does, then we should accept
   // `null`.
 
-  jsg::Unimplemented observe;
-  // Functionality to exert fine-grained control over the fetch, including the ability to cancel
-  // it.
+  jsg::Optional<bool> keepalive;
+  // We do not support keepalive = true but we want to allow code to explicitly set it to the
+  // default (false). Per the spec, keepalive is "a boolean indicating whether or not request can
+  // outlive the global in which it was created."
+
+  jsg::Unimplemented duplex;
+  // TODO(conform):
+  // The duplex option controls whether or not a fetch is expected to send the entire request
+  // before processing the response. The default value ("half"), which is currently the only
+  // option supported by the standard, dictates that the request is fully sent before handling
+  // the response. There are currently a proposal to add a "full" option which is the model
+  // we support. Once "full" is added, we need to update this to accept either undefined or
+  // "full", and possibly decide if we want to support the "half" option. For now we'll throw
+  // if this option is explicitly set since the only value accepted by the standard is not
+  // one that we support.
+
+  jsg::Optional<kj::String> priority;
+  // Specifies the relative priority of the request. We currently do not make use of this
+  // information but we'll go ahead and allow it to be specified since there is no harm
+  // and doing so will help ensure correct behavior later if/when it is implemented. Note
```

Comment:
I guess this means that scripts which specify a priority immediately will eventually see behavior change, which we usually try to avoid.

Reply:
`priority` is already currently defined by the spec as being platform and implementation specific and fetch does not describe any outwardly observable differences beyond *possibly* impacting the timing of a request response. I would strongly expect our implementation of anything, if anything, to generally preserve no-observable-side-effects.